clang/lib/CodeGen/CMakeLists.txt

```diff
@@ -144,6 +144,7 @@ add_clang_library(clangCodeGen
   VarBypassDetector.cpp
 
   DEPENDS
+  vt_gen
```

Comment:
Why it depends?

Reply:
I don't know how that got in here.

Reply:
That change is c3fe727181818d3efdd2ce96598bfe6a2d3ffb83, but I'm not sure how it got squashed into my commit